cli/dcoscli/data/schemas/manifest-schema.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
Is this file used implicitly? I was not able to find how/where this file is used?
@larioj

Reply:
This file may be used for the validation of the manifest created. It fine to be removed.